core/src/main/java/com/backblaze/b2/util/B2Collections.java

```diff
@@ -19,6 +19,10 @@
         return Collections.unmodifiableSet(set);
     }
 
+    public static <T> Set<T> emptySet() {
```

Comment:
Would it be better to make a constant, and re-use the same instance?

Reply:
maybe [Collections.emptySet()](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptySet--) instead?

Reply:
updated

Reply:
sorry.  i wasn't very clear.  maybe the callers could use Collections.emptySet() instead of us making our own wrapper for it.  i think i just made the B2Collections class to hold things I wanted but which weren't available in plain old java -- especially stuff I'd gotten used to having in guava, which we don't use in the sdk.

Reply:
updated